ui/desktop/app/routes/application.js

```diff
@@ -31,11 +31,14 @@ export default class ApplicationRoute extends Route {
   }
 
   /**
-   * Add OS config on controller
+   * Add OS config on controller.
+   * Display OS window actions (minimize, fullscreen, close) for
+   * Windows and Linux. Not enabled for MacOS.
    */
   async setupController(controller) {
     controller.set('isMacOS', await this.ipc.invoke('isMacOS'));
     controller.set('isWindowsOS', await this.ipc.invoke('isWindowsOS'));
+    controller.set('showWindowActions', !controller.isMacOS);
```

Comment:
The controller should probably compute this value, rather than setting it statically here.

Reply:
Updated to use a computed property on the controller.